frontend/src/scenes/surveys/SurveyAppearanceUtils.tsx

```diff
@@ -116,6 +116,10 @@ export function getTextColor(el: never): string {
         hsp = Math.sqrt(0.299 * (r * r) + 0.587 * (g * g) + 0.114 * (b * b))
     return hsp > 127.5 ? 'black' : 'white'
 }
+/** Return the rgb-components for usage within css rgb(). */
+export function getTextColorComponents(color: 'white' | 'black'): string {
+    return color === 'white' ? '255, 255, 255' : '0, 0, 0'
```

Comment:
Text color can only be "black" or "white" at the moment, so I chose a simple function for deriving the rgb-values. This could be changed into a more sophisticated function depending on the allowed values for `textColor`.